types/sharepoint/index.d.ts

```diff
@@ -106,8 +106,8 @@ declare namespace SP {
         static resizeImageToSquareLength(imgElement: HTMLImageElement, squareLength: number): void;
     }
 
-    interface PageContextInfo {
-        new(): PageContextInfoInstance;
+    class PageContextInfo {
+        constructor();
         get_siteServerRelativeUrl(): string;
```

Comment:
I'm confused, were these all on the static side instead of the instance before? Is anyone using `PageContextInfoInstance` anymore?

Reply:
good catch! thanks! I went ahead and did not research, the Object Model is poorly documented for the JSOM but in the js bundles, there are not PageContextInfoInstance as far as I could find. Also looking at the code running, I noticed that the methods of that class are actually static, updated that as well.